definition/data/sheets/CaseEvent.json

```diff
@@ -10,7 +10,7 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "InterlocutoryJudgment", "Name": "Interlocutory Judgment Req", "Description": "Interlucatory Judgment Request", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByDefendant", "Name": "Defendant referred judge", "Description": "Refer To Judge By Defendant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByClaimant", "Name": "Claimant referred judge", "Description": "Refer To Judge By Claimant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "*", "SecurityClassification": "Public"},
```

Comment:
This event needs to have an about to submit callback so the state can be dynamically set